flink-tests/src/test/java/org/apache/flink/test/javaApiOperators/CoGroupITCase.java

```diff
@@ -301,6 +309,51 @@ public void testCoGroupWithMultipleKeyFieldsWithFieldSelector() throws Exception
 	}
 
 	@Test
+	public void testCoGroupWithMultipleKeyFieldsWithFieldSelector2() throws Exception {
```

Comment:
Can you change the method name to `testCoGroupWithRangePartitioning` and move the method to bottom of the file (before the helper classes).